org.grisbi.Grisbi.yml

```diff
@@ -56,7 +58,4 @@ modules:
         path: grisbi.metainfo.xml
     post-install:
       - install -m 644 -p -D grisbi.metainfo.xml /app/share/metainfo/org.grisbi.Grisbi.metainfo.xml
-      - mv /app/share/icons/hicolor/scalable/apps/grisbi.svg /app/share/icons/hicolor/scalable/apps/org.grisbi.Grisbi.svg
-      - mv /app/share/icons/hicolor/scalable/mimetypes/application-x-grisbi.svg /app/share/icons/hicolor/scalable/mimetypes/org.grisbi.Grisbi.application-x-grisbi.svg
```

Comment:
you should have kept line 60. Sadly there is nothing to deal with renaming mime icons automatically.